modules/lazyload/module.php

```diff
@@ -0,0 +1,113 @@
+	const EXPERIMENT_NAME = 'e_bg_lazyload';
+
+	public function get_name() {
+		return 'lazyload';
+	}
+
+	public static function get_experimental_data() {
+		return [
+			'name' => static::EXPERIMENT_NAME,
+			'title' => esc_html__( 'Lazy Load Background Images', 'elementor' ),
+			'tag' => esc_html__( 'Performance', 'elementor' ),
+			'description' => esc_html__( 'Lazy loading images that are not in the viewport improves initial page load performance and user experience. By activating this experiment all background images except the first one on your page will be lazy loaded to improve your LCP score', 'elementor' ),
+			'release_status' => Experiments_Manager::RELEASE_STATUS_BETA,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+		];
+	}
+
+	public function enqueue_scripts() {
+		wp_enqueue_script(
+			'elementor-lazyload',
+			$this->get_js_assets_url( 'lazyload' ),
+			[ 'elementor-frontend' ],
+			ELEMENTOR_VERSION,
+			true
+		);
+	}
+
+	public function enqueue_styles() {
+		wp_enqueue_style(
+			'elementor-lazyload',
+			$this->get_css_assets_url( 'modules/lazyload/frontend' ),
+			[],
+			ELEMENTOR_VERSION
+		);
+	}
+
+	private function update_element_attributes( Element_Base $element ) {
+		$settings = $element->get_settings_for_display();
+		$controls = $element->get_controls();
+		$keys = null;
+
+		$controls_with_background_image = array_filter( $controls, function( $control ) {
+			if ( Utils::get_array_value_by_keys( $control, [ 'background_lazyload', 'active' ] ) ) {
+				return true;
+			}
+		} );
+
+		// For now, it only support the first control within an element that has lazyload enabled,
+		// it will support more controls in the same element that used lazyload in the future.
+		foreach ( $controls_with_background_image as $control_name => $control ) {
+			$keys = Utils::get_array_value_by_keys( $control, [ 'background_lazyload', 'keys' ] );
+			break;
+		}
+
+		if ( $keys ) {
+			$background_image_url = Utils::get_array_value_by_keys( $settings, $keys );
+			if ( $background_image_url ) {
+				$element->add_render_attribute( '_wrapper', [
+					'data-e-bg-lazyload-src' => $background_image_url,
+				] );
+			}
+		}
+	}
+
+	private function reduce_background_image_size( $value, $css_property, $matches, $control ) {
+		if ( Utils::get_array_value_by_keys( $control, [ 'background_lazyload', 'active' ] ) ) {
+			if ( 0 === strpos( $css_property, 'background-image' ) && '{{URL}}' === $matches[0] ) {
+				$thumbnail_image_src = wp_get_attachment_image_src( $value['id'], 'thumbnail' );
+				if ( $thumbnail_image_src ) {
+					$value['url'] = $thumbnail_image_src[0];
+				}
+			}
+		}
+		return $value;
+	}
+
+	public function __construct() {
+		parent::__construct();
+
+		add_action( 'elementor/element/after_add_attributes', function( Element_Base $element ) {
+				$this->update_element_attributes( $element );
+		} );
+
+		add_filter('elementor/files/css/property', function( $value, $css_property, $matches, $control ) {
```

Comment:
```suggestion
		add_filter( 'elementor/files/css/property', function( $value, $css_property, $matches, $control ) {
```

Reply:
Not relevant, the file content is updated.